packages/blade/src/components/Link/BaseLink/BaseLink.tsx

```diff
@@ -187,6 +187,10 @@ const BaseLink = ({
   intent,
   contrast = 'low',
   accessibilityLabel,
+  // @ts-expect-error avoiding exposing to public
+  className,
+  // @ts-expect-error avoiding exposing to public
+  style,
```

Comment:
This will only stop TS projects. Lets not allow overriding styles. The use-case here is not strong enough to allow style overriding. We just want to add spacing in stories and that's not a good reason to expose this 😓

Reply:
Yep totally got your point. 

But I'm not just adding padding. 

```ts
const StyledLink = styled(BaseLink)(({ theme }) => ({
  ...screenReaderStyles,      // screen reader styles
  top: theme.spacing[4],     // padding
  left: theme.spacing[4],     // padding
  ':focus': {    // on focus -> reset screen reader styles
    padding: theme.spacing[1],
    clip: 'auto',
    clipPath: 'unset',
    width: 'auto',
    height: 'auto',
  },
}));
```

Reply:
Can't we do all of these styling on a wrapper on top of Link?

```jsx
const SkipNavContainer = styled.div({
...screenReaderStyles,
})

<SkipNavContainer>
  <Link />
</SkipNavContainer>
```

Reply:
Unfortunately we can't since:

- now the focus will be on the div so we also need to add :focus on the div. 
- then add tabindex={0} to make it focusable
- focus styling will be different 
- Pressing enter will not trigger the link navigation but rather the div will be clicked. 

:(

Reply:
Okay, let's move ahead with this then. I'll take a final review and approve the PR